content/providers/01-ai-sdk-providers/08-amazon-bedrock.mdx

```diff
@@ -368,7 +368,7 @@ const { text, reasoning, reasoningDetails } = await generateText({
   prompt: 'How many people will live in the world in 2040?',
   providerOptions: {
     bedrock: {
-      reasoningConfig: { type: 'enabled', budgetTokens: 1024 },
+      reasoning_config: { type: 'enabled', budgetTokens: 1024 },
```

Comment:
Changes look good to me for the current version. It seems to have been incorrect since the docs where added: https://github.com/vercel/ai/pull/4980/files#diff-7ad7b5eb5101f87657c996a90d248af47e3d0968bca01d4c917273e5a74d486eR334